Cargo.toml

```diff
@@ -22,8 +22,8 @@ pallet-authority-discovery = { git = "https://github.com/PolymathNetwork/substra
 pallet-authorship = { git = "https://github.com/PolymathNetwork/substrate", branch = "polymesh-develop" }
 pallet-babe = { git = "https://github.com/PolymathNetwork/substrate", branch = "polymesh-develop" }
 pallet-balances = { git = "https://github.com/PolymathNetwork/substrate", branch = "polymesh-develop" }
-#pallet-contracts = { git = "https://github.com/PolymathNetwork/substrate", branch = "polymesh-develop" }
-#pallet-contracts-primitives = { git = "https://github.com/PolymathNetwork/substrate", branch = "polymesh-develop" }
+pallet-contracts = { git = "https://github.com/PolymathNetwork/substrate", branch = "polymesh-develop" }
+pallet-contracts-primitives = { git = "https://github.com/PolymathNetwork/substrate", branch = "polymesh-develop" }
 #pallet-contracts-rpc = { git = "https://github.com/PolymathNetwork/substrate", branch = "polymesh-develop" }
 #pallet-contracts-rpc-runtime-api = { git = "https://github.com/PolymathNetwork/substrate", branch = "polymesh-develop" }
```

Comment:
The contracts RPC endpoints should be enabled too.

Reply:
Let's revisit this in a follow up. For now, the PR builds, so let's try to "ship it".